integtests/clients/appsync_client.py

```diff
@@ -122,6 +122,16 @@ def create_opensearch_workspace(self, input):
         )
         return self.client.execute(query).get("createOpenSearchWorkspace")
 
+    def create_kendra_workspace(self, input):
```

Comment:
Can you add these new functions to the following test? https://github.com/aws-samples/aws-genai-llm-chatbot/blob/main/integtests/security/unauthorized_test.py

Reply:
Done